index.html

```diff
@@ -1196,39 +1197,40 @@ <h2><dfn>RTCQuicStreamState</dfn> Enum</h2>
               <td><dfn><code id=
               "idl-def-RTCQuicStreamState.closing">closing</code></dfn></td>
               <td>
-                <p>The procedure to close down the QUIC stream has started.
+                <p>The procedure to close down the <code><a>RTCQuicStream</a></code>
+                <var>stream</var> has started.
                 This can happen in multiple ways:</p>
                   <ol>
-                     <li>A STREAM frame with the FIN flag set has not yet been
-                     received and the <code>reset</code> method has been called.
-                     This causes a RST_STREAM frame to be queued for transmission,
-                     the read and write buffers to be cleared and the
-                     <a>[[\Readable]]</a> and <a>[[\Writeable]]</a> slots are set
+                     <li><var>stream</var> has not <a>finished reading</a> and the
+                     <code>reset</code> method has been called. This causes a
+                     RST_STREAM frame to be queued for transmission, the read
+                     and write buffers to be cleared and <var>stream</var>'s
+                     <a>[[\Readable]]</a> and <a>[[\Writeable]]</a> slots to be set
                      to <code>false</code>.</li>
-                     <li>A STREAM frame with the FIN flag set has not yet been
-                     received and the <code>finish</code> method has been called.
+                     <li><var>stream</var> has not <a>finished reading</a> and the
+                     <code>finish</code> method has been called.
                      This causes a STREAM frame with the FIN flag set to be queued
-                     for transmission, the write buffer to be cleared and the
-                     <a>[[\Writeable]]</a> slot is set to <code>false</code>.</li>
-                     <li> The QUIC stream has received STREAM frame with the FIN flag set.</li>
+                     for transmission, the write buffer to be cleared and <var>stream</var>'s
+                     <a>[[\Writeable]]</a> slot to be set to <code>false</code>.</li>
+                     <li><var>stream</var> has <a>finished reading</a>.</li>
                   </ol>
               </td>
             </tr>
             <tr>
               <td><dfn><code id=
               "idl-def-RTCQuicStreamState.closed">closed</code></dfn></td>
               <td>
-                <p>The QUIC stream has been closed or could not be established.
-                This can happen in multiple ways:</p>
+                <p>The <code><a>RTCQuicStream</a></code> <var>stream</var> has been
+                closed or could not be established. This can happen in multiple ways:</p>
                   <ol>
                      <li>A RST_STREAM frame has been received.</li>
                      <li>A RST_STREAM frame has been sent and a STREAM frame

```

Comment:
This bullet contradicts your change to reset().

Reply:
Change to reset() reverted, so should be consistent now.

Reply:
Change reverted, so should be consistent now.